core/types/types.go

```diff
@@ -94,31 +94,50 @@ type Migration struct {
 type MigrationStatus int
 
 const (
+	// NoActiveMigration indicates there is currently no migration process happening on the network.
 	NoActiveMigration MigrationStatus = iota
-	MigrationNotStarted
+
```

Comment:
Maybe just make these strings constants? You have a `String()` method below, it might be easier to just make it a string from the start. This also helps with forward compatibility in case you want to remove one.